src/ionic/src/lib/ui-ionic.config.ts

```diff
@@ -57,7 +57,7 @@ export const IONIC_FORMLY_CONFIG: ConfigOption = {
     {
       name: 'select',
       component: FormlyFieldSelect,
-      wrappers: ['form-field'],
+      wrappers: [], // TODO: find out why the wrapper doesn't work with an ion-select
```

Comment:
for select add the following style in `FormlyFieldSelect`:
```ts
@Component({
  selector: 'formly-field-ion-select',
  ...,
  styles: [':host { display: inherit; }'],
})
export class FormlyFieldSelect extends IonFieldType {
}
```

Reply:
Awesome thanks!